src/dispatch/stage.rs

```diff
@@ -242,20 +242,42 @@ impl<'a> StagesBuilder<'a> {
         let new_writes = new_writes.into_iter();
 
         let num_groups = ids[stage].len();
+        let mut dep_conflict = false;
 
         let conflict = (0..num_groups)
             .filter(|&group| {
                 let read_and_write = writes[stage][group]
```

Comment:
Wouldn't `read_or_write` be better name for this variable.

Reply:
No, this is an iterator with both read and write ids.

Reply:
Well I associate and with intersection and or with union, so I first read that the iterator has elements that are both write and read. Maybe I am the only one though.

Reply:
Would `reads_and_writes` be clearer for you?

Reply:
Yeah that would be better.